### PR TITLE
tools/mpremote/mpremote: Additonal mpremote disconnect shortcut.

### LINKED ISSUE #11197: Additonal alternative mpremote disconnect keyboard shortcut
On a german MacBook keyboard, maybe on others as well, I can't use `ctrl-]` to close the `mpremote` session, see https://github.com/micropython/micropython/blob/f34af3e42e6e823f5fbff1f8e729fe14460ea4ca/tools/mpremote/mpremote/repl.py#L11

I would suggest to additionally support `ctrl-x` as done in [rshell](https://github.com/dhylands/rshell)

```python
if c in [b"\x1d", b"\x18"]:  # ctrl-] or ctrl-x, quit
    break
```

I'm of course willing to implement this complex change. I just wanted to make sure we are on the same page and I haven't missed anything else.

### PR DESCRIPTION
**Pull request by brainelectronics**
`mpremote` REPL can be closed with `ctrl+]` or `ctrl+x`, resolves #11197

### CODE REVIEW

File: tools/mpremote/mpremote/repl.py [hunk 1]
```diff
@@ -8,7 +8,7 @@ def do_repl_main_loop(state, console_in, console_out_write, *, code_to_inject, f
         console_in.waitchar(state.pyb.serial)
         c = console_in.readchar()
         if c:
-            if c == b"\x1d":  # ctrl-], quit
+            if c in [b"\x1d", b"\x18"]:  # ctrl-] or ctrl-x, quit
```
Comment (dpgeorge) [inline tools/mpremote/mpremote/repl.py:11]: Please use a tuple here instead of a list.

**Comment by dpgeorge** [comment]
I think it would be good to also update the banner info that currently says `Use Ctrl-] to exit this shell`, so it also mentions the new ctrl-x.

**Comment by brainelectronics** [comment]
Thank you @dpgeorge for pointing out. I've updated my change. The workflow `unix port/qemu_arm` is most likely not failing due to my recent change as far as I can see from the builds on master

**Comment by dpgeorge** [comment]
Thanks for updating.

**Pull request merged**